frontend/integration-tests-cypress/cypress/support/index.ts

```diff
@@ -0,0 +1,18 @@
+import './commands';
+
+// Must be declared global to be detected by typescript (allows import/export)
+// eslint-disable @typescript/interface-name
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
```

Comment:
This is interesting, curious about how it enforces chainable when the actual functions don't `return this`.

Reply:
Thinking about organization again, the growth that comes with adding tests.  Wondering if we can namespace this a bit to help keep it tidy and reduce the changes of writing the same function twice down the road:

```ts
Cypress {
  selector {
     element()
     action()
  }
  project {
    create()
    delete()
  }
  nav {
    clickLink()
  }
  detailsPage {}
  listPage {}
  // etc.
}
```

Reply:
Fixed/adjusted in latest update